examples/go/hello-app-bar/hello.go

```diff
@@ -0,0 +1,170 @@
+package main
+
+import (
+	_ "embed"
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/mattermost/mattermost-plugin-apps/apps"
+	"github.com/mattermost/mattermost-plugin-apps/apps/appclient"
+	"github.com/mattermost/mattermost-plugin-apps/utils/httputils"
+)
+
+//go:embed icon.png
+var IconData []byte
+
+// Manifest declares the app's metadata. It must be provided for the app to be
+// installable. In this example, the following permissions are requested:
+//   - Create posts as a bot.
+//   - Add icons to the channel header that will call back into your app when
+//     clicked.
+//   - Add a /-command with a callback.
+var Manifest = apps.Manifest{
```

Comment:
I'll be moving this example to https://github.com/mattermost/mattermost-app-examples